packages/admin/admin/src/tabs/Tab.tsx

```diff
@@ -0,0 +1,99 @@
+import { Info } from "@comet/admin-icons";
+import { Box, ComponentsOverrides, IconProps, Theme, Tooltip, TooltipProps, Typography } from "@mui/material";
+import MuiTab, { TabProps as MuiTabProps } from "@mui/material/Tab";
+import { WithStyles, withStyles } from "@mui/styles";
+import * as React from "react";
+
+import { Status, StatusBadge } from "../statusBadge/StatusBadge";
+import { styles, TabClassKey } from "./Tab.styles";
+
+export interface TabProps extends Omit<MuiTabProps, "children" | "icon" | "iconPosition"> {
+    label: React.ReactNode;
+    forceRender?: boolean;
+    children: React.ReactNode;
+    currentTab: number;
+    tabIcon?: React.FunctionComponentElement<IconProps>;
+    status?: Status;
+    showStatusIcon?: boolean;
+    statusIcon?: React.FunctionComponentElement<IconProps>;
+    showTooltip?: boolean;
+    tooltipMessage?: string;
+    tooltipIcon?: React.FunctionComponentElement<IconProps>;
+    tooltipPlacement?: TooltipProps["placement"];
+    smallTabText?: boolean;
```

Comment:
Regarding the props, I'd stay with the terms UX uses: 

<img width="770" alt="Bildschirmfoto 2023-12-27 um 11 00 03" src="https://github.com/vivid-planet/comet/assets/13380047/ee525972-0826-49e9-be04-93004f527036">

So 
`tabIcon` -> `icon`
`tooltipIcon` -> `helperIcon`

I'd further:
remove `showTooltip` (should be controlled by `helperIcon`)
omit `tooltipMessage` and `tooltipPlacement` in favor of passing a `Tooltip` via `helperIcon` (Alternative: only pass `helperText`. Omit `helperIcon`, always show the `Info` icon instead)
rename `smallTabText` to `size`

Reply:
Alright. @johnnyomair does this align with your expectations?

Reply:
Should `size` be a number then or an enum value or what? Because at the moment it's a boolean and the name of a boolean variable (or prop in this case) should indicate that it is a boolean.